### ISSUE TITLE
Display static themes in addon queues to static theme reviewers

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons-server#7750]
Part of #7385 
Static themes are add-ons so will be displayed in the add-on review queues - new addons and updates.  But they should only be shown to reviewers who have appropriate permissions - and non-static-theme add-ons hidden from reviewers who only review static themes.

**Comment by AlexandraMoga** [comment]
@eviljeff could you please explain which permissions you are referring to when saying "appropriate permissions"?
I thought that users with legacy add-on reviewer permissions will have visibility to static themes since they are listed in the Legacy queues (New Add-ons, Add-on Updates)
However, I was able to see static themes in these queues only when logging in with an Admin user.

**Comment by eviljeff** [comment]
I added a new permission (`Addons:ThemeReview`)- but haven't yet created any groups that use it.  I'll check with Product if they want an existing group changed or a new one created.

So you can't fully test this right now - you can test that Static Themes don't show up for legacy-only reviewers but you can't check that only Static Themes show up (and not legacy addons) for users that are only supposed to review Static Themes

**Comment by AlexandraMoga** [comment]
@eviljeff in that case, would it make sense to test this feature as an Admin user (since is the only one who has visibility to Static Themes queue) or should we consider it blocked until the relevant group for the `Addons:ThemeReview` gets settled?

**Comment by eviljeff** [comment]
Admin users should be able to see Static Themes in those queues.  But they'll also seem legacy addons, which a pure static theme reviewer wouldn't see.

**Comment by eviljeff** [comment]
so blocked-ish

**Comment by eviljeff** [comment]
#7769 to add the permissions - though we might not do that for a few weeks so Persona reviewers aren't confused by half-finished and partially functional tools suddenly appearing.

**Comment by AlexandraMoga** [comment]
@eviljeff since this issue is only partially testable, can we suggest to postpone the milestone or mark it as blocked by #7769 ?
I'd like to be able to test the full functionality of the static themes queue before marking this issue as verified.
Does this work for you?

**Comment by eviljeff** [comment]
Does marking this issue as verified and re-checking queue visibility as part of #7769 work for you?

**Comment by AlexandraMoga** [comment]
I'm going to mark this as verified for the following features:
- Static Theme queue available in Reviewer's dashboard - visible to Admins only **[1]**
- Static Theme extension 'type' added in New Add-ons/Add-on updates queues and in Advanced search - visible to Admins only **[2]**
- Static Theme queue not available to users with Legacy reviewer permissions (i.e. New Add-ons/Add-on Updates) queues **[3]**

**[1]**
![image](https://user-images.githubusercontent.com/31961530/37454744-06a55a94-2844-11e8-96d5-bd166c20b807.png)

**[2]**
![image](https://user-images.githubusercontent.com/31961530/37454768-1446a856-2844-11e8-8a0a-f1b386f8fa8c.png)

**[3]**
![image](https://user-images.githubusercontent.com/31961530/37454787-24fae694-2844-11e8-9ad8-3d0b57ea10db.png)

The full functionality will be tested when #7769 lands.
